packages/next/src/server/patch-error-inspect.ts

```diff
@@ -27,9 +29,21 @@ function frameToString(frame: StackFrame): string {
   if (frame.column !== null && sourceLocation !== '') {
     sourceLocation += `:${frame.column}`
   }
+
+  const filePath =
+    frame.file !== null &&
+    frame.file.startsWith('file://') &&
+    URL.canParse(frame.file)
+      ? // If not relative to CWD, the path is ambiguous to IDEs and clicking will prompt to select the file first.
+        // In a multi-app repo, this leads to potentially larger file names but will make clicking snappy.
+        // There's no tradeoff for the cases where `dir` in `next dev [dir]` is omitted
+        // since relative to cwd is both the shortest and snappiest.
+        path.relative(process.cwd(), new URL(frame.file).pathname)
```

Comment:
does this work on windows? you could use this instead:

```suggestion
        path.relative(process.cwd(), url.fileURLToPath(frame.file))
```

Reply:
Why wouldn't it? But the suggested method looks clearer so I pick that.

Reply:
TIL. This isn't actually equivalent but `fileURLToPath` is the right cross-platform choice: https://nodejs.org/api/url.html#urlfileurltopathurl-options